### DISCUSSION TITLE
[Android] Showing page loading indicator on tabbed page

### DISCUSSION THREAD

**Discussion by brunck** [MvvmCross/MvvmCross#4123] [Q&A]
I've got a problem in Xamarin.Forms on Android where I'm trying to show a page loading indicator (using Acr.UserDialogs) on a tabbed page, but absolutely all the ViewModel and page events happen before the default tab is loaded, on both the tabbed page and the tab itself. I can't seem to hook into anything that happens after the page is fully displayed.

I am using an MvxListView on the page, but I'm not seeing any events I can use there to start showing and then hiding a loading indicator screen.

Is there something I can do, or am I out of luck on Android?

**Comment by Hackmodford** [comment]
Does the `ViewCreated()` virtual method not work?

**Reply by brunck** [reply]
No that happens way before my tab is displayed. It doesn't actually draw the list view until well after all the appearing, appeared etc. lifecycle methods have happened. Xamarin.Forms and Android seem to have trouble playing nice around the page lifecycle and I suspect MvvmCross is affected by that.

**Reply by Hackmodford** [reply]
Since they're tabs, they probably all technically get created at the same time. I'm guessing this means by the time you get to your tab the data is already there?
